packages/driver/src/multi-domain/errors.ts

```diff
@@ -0,0 +1,31 @@
+import type { $Cy } from '../cypress/cy'
+import $errUtils, { ErrorFromProjectRejectionEvent } from '../cypress/error_utils'
+
+export const handleErrorEvent = (cy: $Cy) => {
+  return (handlerType: string, frameType?: string) => {
+    return (event) => {
+      const { originalErr, err, promise } = $errUtils.errorFromUncaughtEvent(handlerType, event) as ErrorFromProjectRejectionEvent
+      const handled = cy.onUncaughtException({
+        err,
+        promise,
+        handlerType,
+        frameType: frameType ?? 'app',
```

Comment:
Shouldn't matter, but I don't think we want to coalesce here just for error handling safety in the primary
```suggestion
        frameType: frameType || 'app',
```

Reply:
Decided to make this required so this is no longer needed.